package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "flex-layout-srcs",
   "description": "Angular Flex Layout",
-  "version": "13.0.0-beta.38",
+  "version": "14.0.0-beta.38",
```

They usually increment the beta version on new releases.  
```suggestion
  "version": "14.0.0-beta.39",
```